src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -148,6 +148,17 @@ internal LuceneSearchUtility LuceneUtility
             }
         }
 
+        /// <summary>
+        /// Return a dictionary of GraphChecksumItems.
+        /// Key will be the workspace guid and its value will be a list of saved checksums(sha256 hash) for that workspace.
+        /// </summary>
+        internal Dictionary<string, List<string>> GraphChecksumDictionary { get; set; }
```

Comment:
can you explain why we need both of these data structures?

Reply:
ah, seems like if you used JSON you could just serialize this directly and skip the list.

Reply:
Yes using JSON serialization now. https://github.com/DynamoDS/Dynamo/pull/15116